editoast/src/views/authz.rs

```diff
@@ -145,6 +152,126 @@ pub(in crate::views) async fn user_groups(
     Ok(Json(result))
 }
 
+#[derive(Deserialize, Clone, ToSchema)]
+pub struct UsersInfoRequest {
+    #[serde(default)]
+    ids: Vec<i64>,
+    #[serde(default)]
+    identities: Vec<String>,
+}
+
+#[derive(Serialize, ToSchema)]
+pub struct UserInfo {
+    id: i64,
+    name: String,
+    identities: Vec<String>,
+    roles: HashSet<Role>,
+    #[schema(inline)]
+    groups: HashSet<Group>,
```

Comment:
```suggestion
    groups: HashMap<i64, String>, // id => name
```

the schema would be nicer maybe?

Reply:
nope, that's not valid json...